pkg/substitution/substitution.go

```diff
@@ -49,6 +49,9 @@ func FillConfigValues(config config.OperatorConfig, templates []client.Object) [
 		switch obj := templateCopy.(type) {
 		case *appsv1.Deployment:
 			obj.Spec.Template.Spec = setCloudControllerImage(config, obj.Spec.Template.Spec)
+			if config.IsSingleReplica {
+				obj.Spec.Replicas = &[]int32{1}[0]
```

Comment:
Replicas is a slice? I thought this should just be an int pointer?

Reply:
It's a semi-idiomatic way to get a pointer to the integer (kind of &1). Here we create a slice of int32 with just one element, and then we get a pointer to the first element.

Reply:
That's quite hard to understand IMO, would prefer to use something like https://pkg.go.dev/k8s.io/utils/pointer#Int32 which I know we are using elsewhere for this purpose

Reply:
thanks! this is a more convenient way get int pointers